Indico/Storage/UploadStream.cs

```diff
@@ -42,11 +39,11 @@ async public Task<JArray> Call()
                 fileParameters.Add(param);
             }
 
-            MultipartFormUpload formUpload = new MultipartFormUpload(this._client)
+            var formUpload = new MultipartFormUpload(_client)
             {
                 FileParameters = fileParameters
             };
-            JArray uploadResult = await formUpload.Call();
+            var uploadResult = await formUpload.Call();
 
             foreach (JObject uploadMeta in uploadResult)
```

Comment:
switch to var here also?

Reply:
If I use var in line 51 it gets resolved with `JToken` instead of `JObject` which breaks the code.